src/doc/template/publish.js

```diff
@@ -406,18 +406,6 @@ function buildNav(members) {
 
   }
 
-  if (members.events.length) {
-
-    members.events.forEach(function(e) {
-      if (!hasOwnProp.call(seen, e.longname)) {
-
-        nav.event.members.push(linkto(e.longname, e.longname.replace("module:", "")));
-      }
-      seen[e.longname] = true;
-    });
-
-  }
```

Comment:
I'm not sure if this is the best way to handle this, but this removes the Brush events that were inadvertently added to the API navigation dropdown menu in #566:

<img width="290" alt="screen shot 2018-04-04 at 9 12 53 pm" src="https://user-images.githubusercontent.com/847532/38342566-349f8b72-384d-11e8-82cd-dfbb0a7ba24e.png">

Reply:
To be honest, I have no idea how this templating works either. But we do not need those. Could you attach a screenshot of this and other changes? how does demo look like now?

Reply:
Here is what the API navigation dropdown menu looks like now:

<img width="228" alt="screen shot 2018-04-04 at 10 11 56 pm" src="https://user-images.githubusercontent.com/847532/38344019-46217984-3855-11e8-8ec2-c26eef8babdd.png">

The Brush Chart demo functionality change is illustrated in the gif above.

Reply:
Ha, I don't have any idea either, this was tunned by another contributor.